controllers/renovate_util.go

```diff
@@ -173,17 +206,23 @@ func (u ComponentDependenciesUpdater) GetUpdateTargetsBasicAuth(ctx context.Cont
 			username = DefaultRenovateUser
 		}
 
+		customRenovateOptions, err := u.ReadCustomRenovateConfigMap(ctx, &component)
+		if err != nil {
+			log.Error(err, "failed to read custom renovate config map", "ComponentName", component.Name, "ComponentNamespace", component.Namespace)
```

Comment:
Do we just ignore the error?

Reply:
well we are logging it

we could fail here, but I'd rather just actually perform nudging

user config is just additional thing

Reply:
I see, in such case please add a comment and improve error message to show that nudge will still happen.

Reply:
ok